src/CommunityToolkit.Maui/Layouts/StateLayout/StateLayout.shared.cs

```diff
@@ -0,0 +1,143 @@
+﻿using CommunityToolkit.Maui.Core;
+
+namespace CommunityToolkit.Maui.Layouts;
+
+/// <summary>
+/// The <see cref="StateLayout"/> control enables the user to turn any layout element like a <see cref="Grid"/> or <see cref="StackLayout"/> into an individual state-aware element.
```

Comment:
Let's not use the deprecated `StackLayout` here (;

```suggestion
/// The <see cref="StateLayout"/> control enables the user to turn any layout element like a <see cref="Grid"/> or <see cref="VerticalStackLayout"/> into an individual state-aware element.
```